ckan/views/user.py

```diff
@@ -28,6 +28,12 @@
 user = Blueprint(u'user', __name__, url_prefix=u'/user')
 
 
+@user.errorhandler(403)
```

Comment:
We need to handle other exceptions as well, as currently any error returns a blank server page (400, 404, 500, etc). I think it's safer to explicitly add a decorator for each of the status codes returned by the blueprints.